CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.22.0] - 3-Dec-2020
+
+**Milestone**: Catapult-server finality(0.10.0.4)
+ Package  | Version  | Link
+---|---|---
+SDK Core| v0.22.0 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
+Catbuffer | v0.0.24 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Client Library | v0.10.4  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+
+- **[BREAKING CHANGE]** `Deadline.create` requires the configurable `epochAdjustment` from the network properties.
```

Comment:
would be better to mention epochAdjustment can be retrieved from `network/properties` endpoint or using the repositoryFactory.